api/events/src/main/java/io/opentelemetry/api/events/EventEmitter.java

```diff
@@ -35,18 +36,30 @@ public interface EventEmitter {
   /**
    * Emit an event.
    *
-   * @param eventName the event name, which acts as a classifier for events. Within a particular
-   *     event domain, event name defines a particular class or type of event.
-   * @param attributes attributes associated with the event
+   * @param eventName the event name, which defines the class or type of event. Events names SHOULD
+   *     include a namespace to avoid collisions with other event names.
+   * @param payload the eventPayload, which is expected to match the schema of other events with the
+   *     same {@code eventName}.
+   */
+  void emit(String eventName, AnyValue<?> payload);
+
+  /**
+   * Emit an event.
+   *
+   * @param eventName the event name, which defines the class or type of event. Events names SHOULD
+   *     include a namespace to avoid collisions with other event names.
+   * @param payload the eventPayload, which is expected to match the schema of other events with the
+   *     same {@code eventName}.
    */
-  void emit(String eventName, Attributes attributes);
+  default void emit(String eventName, Map<String, AnyValue<?>> payload) {
```

Comment:
Are you intentionally avoiding the `emit(name)` with no payload? Since the payload is optional, this seems like a nice overload to have.

Reply:
Yeah I think we should add that. Its technically possible via `eventEmitter.build(name).emit()`, but this is helpful.